src/components/Search/Search.tsx

```diff
@@ -274,6 +274,28 @@ const SearchResults: React.FC<{
               </>
             }
           />
+          {searchQuery.filters.length > 0 ? (
+            <Button
+              id="clear-filters-button"
+              buttonType="secondary"
+              type="reset"
+              onClick={() => {
+                changeFilters([]);
+              }}
+              __css={{
```

Comment:
If you need a version of the "secondary" button that is blue, perhaps this is worth asking if the Design System should have one or have a way of creating a blue "secondary" button.

Reply:
spoke to Marty, this variant is in the works for Reservoir and will be included in the next release on Thursday.